source/codegen/metadata/nidmm/functions.py

```diff
@@ -1345,302 +1798,373 @@
+                'cppName': 'vi',
                 'direction': 'out',
+                'grpc_type': 'nidevice_grpc.Session',
                 'name': 'vi',
                 'type': 'ViSession'
             },
             {
+                'cppName': 'errorMessage',
                 'direction': 'out',
-                'get_last_error': 'get_last_error_message',
+                'get_last_error': 'deprecated',
+                'grpc_type': 'string',
                 'name': 'errorMessage',
                 'type': 'ViChar[]'
             }
         ],
         'returns': 'ViStatus'
     },
     'Initiate': {
+        'codegen_method': 'public',
         'parameters': [
             {
+                'cppName': 'vi',
                 'direction': 'in',
+                'grpc_type': 'nidevice_grpc.Session',
                 'name': 'vi',
                 'type': 'ViSession'
             }
         ],
         'returns': 'ViStatus'
     },
     'InvalidateAllAttributes': {
+        'codegen_method': 'public',
         'parameters': [
             {
+                'cppName': 'vi',
                 'direction': 'in',
+                'grpc_type': 'nidevice_grpc.Session',
                 'name': 'vi',
                 'type': 'ViSession'
             }
         ],
         'returns': 'ViStatus'
     },
     'IsOverRange': {
+        'codegen_method': 'public',
         'parameters': [
             {
+                'cppName': 'vi',
                 'direction': 'in',
+                'grpc_type': 'nidevice_grpc.Session',
                 'name': 'vi',
                 'type': 'ViSession'
             },
             {
+                'cppName': 'measurementValue',
                 'direction': 'in',
+                'grpc_type': 'double',
                 'name': 'measurementValue',
                 'type': 'ViReal64'
             },
             {
+                'cppName': 'isOverRange',
                 'direction': 'out',
+                'grpc_type': 'bool',
                 'name': 'isOverRange',
                 'type': 'ViBoolean'
             }
         ],
         'returns': 'ViStatus'
     },
     'IsUnderRange': {
+        'codegen_method': 'public',
         'parameters': [
             {
+                'cppName': 'vi',
                 'direction': 'in',
+                'grpc_type': 'nidevice_grpc.Session',
                 'name': 'vi',
                 'type': 'ViSession'
             },
             {
+                'cppName': 'measurementValue',
                 'direction': 'in',
+                'grpc_type': 'double',
                 'name': 'measurementValue',
                 'type': 'ViReal64'
             },
             {
+                'cppName': 'isUnderRange',
                 'direction': 'out',
+                'grpc_type': 'bool',
                 'name': 'isUnderRange',
                 'type': 'ViBoolean'
             }
         ],
         'returns': 'ViStatus'
     },
-    'LockSession': {
```

Comment:
LockSession was private before... did we use it? I think removing completely is ok.